[package] error if we try to mock a module in 3.6

suo opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#61469**

This feature is not supported, error out early.

Differential Revision: [D29639797](https://our.internmc.facebook.com/intern/diff/D29639797)

suo committed:
[package] error if we try to mock a module in 3.6

This feature is not supported, error out early.

[ghstack-poisoned]

suo commented:
@suo has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D29639797).

Lilyjjo commented on torch/package/package_exporter.py:
```diff
@@ -706,6 +707,11 @@ def mock(
                 If ``allow_empty=True``, no such exception is thrown.
 
         """
+        if version_info < (3, 6):
```
should this be <= or (3,7)?

suo replied:
yes! DAMMIT github copilot failed me

suo committed:
Update on "[package] error if we try to mock a module in 3.6"


This feature is not supported, error out early.

Differential Revision: [D29639797](https://our.internmc.facebook.com/intern/diff/D29639797)

[ghstack-poisoned]

suo commented:
@suo has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D29639797).
